_includes/contributor/labels.md

```diff
@@ -101,7 +101,7 @@ Labels applied to issues through verification and completion. For details on the
 
 All [contributions to DevDocs](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md) receive the following labels:
 
-* `New topic`- New file submissions for content that has never existed on devdocs
-* `Major update` - Significant updates to existing content, such as a new section or example
-* `Technical` - Updates to the code or processes that alter the technical content of the document
-* `Editorial` - Fixes for typos, grammatical inconsistencies, or minor rewrites to correct inaccuracies.
+* `New topic`- New topic submissions for content that has never existed on DevDocs such as tutorials, references, instructions, and so on
+* `Major update` - Significant updates to existing content, such as a new section or examples with sample code
```

Comment:
I don't think adding a section should necessarily count as a major update. We've had some inconsequential sections added that, by this definition, count as major.

Reply:
@keharper Can you suggest a different description

Reply:
Significant detailed new content that affects one or more existing topics.